.travis.yml

```
@@ -10,16 +10,18 @@ jobs:
     # Latest image build/push
     - if: branch = master
       script:
+        - echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
         - make latest && make test
       deploy:
         provider: script
         script: echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin && make push-latest
```

Can you add something like 
```suggestion
        script: 'if [ "$TRAVIS_PULL_REQUEST" = "false" ]; then echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin && make push-latest; fi'
```
so that opening a PR doesn't push `latest`.

doc: https://docs.travis-ci.com/user/pull-requests/